### ISSUE TITLE
Enhance website A11y

### ISSUE THREAD

**Issue by Muhnad** [atom/github#832]
I notice the [githb-atom website](https://github.atom.io/) is not accessible for all users.

the navbar inside the `main.features` have a list of spans that's is when you click on the video above change but as you know the span element is not focusable so you should use `<button>` instead of `<span>` to make it accessible for keyboard users.

also, the website a11y need to more work I notice also the icons do not have `aria-hidden=true` or `aria-label` whatever you want to the screen reader user read.

**Comment by BinaryMuse** [comment]
Thanks for reporting this. I've moved it to the `atom.io` repo at https://github.com/atom/atom.io/issues/119
